Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs

```diff
@@ -25,7 +25,7 @@ internal override void PopulateFromDto(DeviceDto dto)
             // Extract the "standard" attributes from the Attributes dictionary into their own properties
             if (dto.Attributes is not null)
             {
-                if (dto.Attributes.ContainsKey("currentTemperature")) CurrentTemperature = dto.Attributes["currentTemperature"].GetInt32();
+                if (dto.Attributes.ContainsKey("currentTemperature")) CurrentTemperature = dto.Attributes["currentTemperature"].GetDecimal();
```

Comment:
See above comment and use `GetDouble()` instead,

Reply:
Changed!